contracts/common/tokens/MaticWETH.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.5.2;
+
+import { WETH } from "./WETH.sol";
+
+
+contract MaticWETH is WETH {
+  string public name = "Wrapped Ether";
+  string public symbol = "WETH";
+  uint8  public decimals = 18;
+
+  function deposit() public payable {
+    _mint(msg.sender, msg.value);
+    emit Deposit(msg.sender, msg.value);
+  }
+
+  function withdraw(uint wad) public {
+    require(balanceOf(msg.sender) >= wad);
+    _burn(msg.sender, wad);
+    msg.sender.transfer(wad);
+    emit Withdrawal(msg.sender, wad);
+  }
+
+  function withdraw(uint wad, address user) public {
+    require(balanceOf(msg.sender)>= wad);
+    // cast to address payable
+    address(uint160(user)).transfer(wad);
```

Comment:
uint160?

Reply:
@atvanguard instead of this hack I'd prefer `withdraw(uint wad, address payable user)`

Reply:
@jdkanani That was required to cast to `address payable` from `address`. @0xAshish Ok, I am changing the func sig.